code/modules/networks/computer3/mainframe2/utilities.dm

```diff
@@ -1402,3 +1408,10 @@
 
 		mainframe_prog_exit
 		return
+/datum/computer/file/mainframe_program/utility/pwd
+	name = "pwd"
+	size = 1
+	initialize(var/initparams)
+		message_user(read_user_field("curpath"))
+		mainframe_prog_exit
+	return
```

Comment:
not needed / won't work / might not compile

Reply:
oof. will fix

Reply:
right, i've got rid of the random return